packages/flutter/lib/src/material/button_style_button.dart

```diff
@@ -86,6 +86,10 @@ abstract class ButtonStyleButton extends StatefulWidget {
     required this.clipBehavior,
     this.statesController,
     this.isSemanticButton = true,
+    @Deprecated(
+      'Remove this parameter as it is now ignored. '
```

Comment:
Can we add more context to the message here? This should probably mention ButtonStyle and the styleFrom methods where folks can use this now.

Reply:
Great suggestion! I added a line about `ButtonStyle.iconAlignment` alternative and update version tag.